src/loki/search.cc

```diff
@@ -301,12 +311,13 @@ struct bin_handler_t {
         float angle =
             tangent_angle(index, candidate.point, info.shape(),
                           GetOffsetForHeading(edge->classification(), edge->use()), edge->forward());
+        auto z_level = candidate.edge_info->z_level();
```

Comment:
why does this use `candidate.edge_info` and not `info` as is used above? should everything be using `candidate.edge_info`?

Reply:
Yeah, I think we should be using `candidate.edge_info` everywhere - it allows to get rid of additional call of `GraphTile::edgeinfo` we have here.

Reply:
Well, actually it should be `GraphTile::edgeinfo` - we iterate here edge by edge and `candidate.edge_info` would be wrong here - please, forget my previous comment 😄